src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -2275,7 +2275,7 @@ internal static string ConcatenateStringPathArguments(CommandElementAst stringAs
                     {
                         if (parameterName.Equals("MemberName", StringComparison.OrdinalIgnoreCase))
                         {
-                            NativeCompletionMemberName(context, result, commandAst, boundArguments?[parameterName]);
+                            NativeCompletionMemberName(context, result, commandAst, boundArguments?[parameterName], propertiesOnly: false);
```

Comment:
`ForEach-Object` can call method with `-MemberName`, but only if the method doesn't take any arguments. Returning all methods doesn't seem to be very useful.

Reply:
You can actually call methods with arguments using the ArgumentList parameter: `"Hello" | foreach -MemberName contains -ArgumentList 'e'`

Reply:
😮 I'm the one implemented the `-MemberName` feature and I totally forgot the `-ArgumentList` parameter 🤷